pom.xml

```diff
@@ -640,7 +635,7 @@
             <dependency>
                 <groupId>org.glassfish.jersey.core</groupId>
                 <artifactId>jersey-common</artifactId>
-                <version>3.0.15</version>
+                <version>3.1.8</version>
```

Comment:
Requires investigation.

Reply:
I bounced [`US957002`](https://internal.almoctane.com/ui/entity-navigation?p=131002/6001&entityType=work_item&id=957002) over this.